Ansible/play_file

```diff
@@ -0,0 +1,91 @@
+---
+- name: Install To Do App on new webserver
+  hosts: managed_hosts
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: api_key
+      prompt: What is the api key?
+      private: no
+
+    - name: api_token
+      prompt: What is the api token?
+      private: no
+
+    - name: trellobid
+      prompt: What is the trellobid?
+      private: no    
+
+  tasks: 
+  - name: Install git on VM
+    ansible.builtin.yum:
+      name: git
+      state: present
+    become: yes
+
+  - name: Install Python on VM
+    ansible.builtin.yum:
+      name: python
+      state: latest
+    become: yes
+
+  - name: Check Poerty on VM
+    ansible.builtin.shell:
+      cmd: PATH=$PATH:/etc/poetry/bin; which poetry
+    register: poetry_exists
+    failed_when: no
+    changed_when: no
+
+  - name: Install Poetry on VM
+    ansible.builtin.shell:
+      cmd: curl -sSL https://install.python-poetry.org | POETRY_HOME=/etc/poetry python3 -
+    when: poetry_exists.rc != 0
+    become: yes    
+ 
+  - name: Create base directory for To Do App
+    ansible.builtin.file: 
+      path: /opt/todoapp 
+      state: directory 
+      owner: ec2-user 
+      mode: 0774
+      recurse: yes
+    become: yes
+
+  - name: Get the repo from GIT 
+    ansible.builtin.git:
+     repo: 'https://github.com/nasarullah-sheikh-uk/Module3Exercise.git'
+     dest: /opt/todoapp
+     force: yes
+     version: M4Exercise
+
+  - name: Install Flask
+    ansible.builtin.shell:
+      cmd: cd /opt/todoapp; /etc/poetry/bin/poetry install
```

Comment:
Again mild preference rather than strict advantage, but [the `chdir` argument to change directory](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/shell_module.html#parameter-chdir) can read a little nicer than combining commands